ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -86,7 +86,15 @@ function unconvertPointCoordinate(point) {
 }
 
 function offMap([longitude, latitude]) {
-  return longitude < -180 || longitude > 180 || latitude < -90 || latitude > 90
+  const normalizedLongitude =
```

Comment:
✏️ Extract the normalization process into a function since it is being repeated.